src/preprocessing.ml

```diff
@@ -635,6 +616,24 @@ let action_for_pp sctx ~dep_kind ~loc ~expander ~action ~src ~target =
   | None -> action
   | Some dst -> Action.with_stdout_to (Path.build dst) action
 
+(* Generate rules for the reason modules in [modules] and return a
```

Comment:
This comment should be generalised since we now support more than just the `reason` dialect

Reply:
Done.